app.js

```diff
@@ -0,0 +1,17 @@
+const form = document.querySelector('form');
+const input = document.querySelector('input');
+const ul = document.createElement('ul');
+const li = document.createElement('li');
+
+addTask = (e) => {
```

Comment:
I recommend you to keep functions _pure_ 
[Article 1](https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-pure-function-d1c076bec976)